### ISSUE TITLE
False positive: `XmlNodeList.Should().HaveCount(1)`

### ISSUE THREAD

**Issue by csoltenborn** [fluentassertions/fluentassertions.analyzers#82]
### Description

Analyzer suggests to use an assertion that does not exist.

![image](https://user-images.githubusercontent.com/352576/46571082-08251180-c96f-11e8-9702-3cee2ae64a44.png)

### Complete minimal example reproducing the issue

```csharp
using System.Xml;
using FluentAssertions;

namespace Foo
{
    class Bar
    {
        void Baz()
        {
            XmlNodeList childNodes = new XmlDocument().ChildNodes;
            childNodes.Should().HaveCount(1);
            // childNodes.Should().ContainSingle();
        }
    }
}
```

### Versions

FluentAssetions 0.11.4
.NET 4.5
